ls1intum/Artemis
# `General`: Remove participant score pages

### Checklist
#### General


#### Server
#### Client

### Motivation and Context



The _Participant Score_ pages, for a course or an exam, do not offer any real uses cases or benefits for instructors (see discussion in #6230). They are mainly for debugging, as the student scores are shown on another page.
Example:
- https://artemis-test2.artemis.in.tum.de/course-management/174/participant-scores
- https://artemis-test2.artemis.in.tum.de/course-management/174/exams/65/participant-scores

### Description


Removed the _Participant Score_ pages for courses and exams.

Decluttered the Course Management header by grouping some less important buttons into a dropdown.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Check the 'Participant Scores' button is no longer there
4. Ensure everything else, especially the 'Scores' page, is still working as expected

#### Exam Mode Testing



Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Go to an existing exam with student submissions (or create one)
3. Ensure that the 'Scores' page still works as expected

### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage



_Unchanged._

### Screenshots



<img width="600" src="https://user-images.githubusercontent.com/102237363/218337579-88fe4a2e-33c6-4495-94a0-d8e5ec9b9d3a.png">

<img width="600" src="https://user-images.githubusercontent.com/102237363/222353161-3f87ed53-d9df-4519-86a3-4fc014e7489d.png">
